src/components/project/sections/the-challenge.tsx

```diff
@@ -1,27 +1,222 @@
 import Image from "next/image";
 import { DynamicTextAndImage } from "../../../interfaces/project";
+import { Row } from "../../global/grid/Row";
+import { Col } from "../../global/grid/Col";
+import Pill from "../../global/pill";
+import styled from "@emotion/styled";
+import { useState } from "react";
+import { dimensions, breakpoints, colors } from "../../../styles/variables";
+import { getRemSize } from "../../../styles/globalCss";
+import useIsDesktop from "../../../hooks/useIsDesktop";
+import useIsTablet from "../../../hooks/useIsTablet";
+import { motion } from "framer-motion";
 
 interface IProps {
   content: DynamicTextAndImage[];
 }
 
+const StyledBigWrapper = styled.div`
+  position: relative;
+`;
+
+const StyledPillWrapper = styled.div``;
```

Comment:
If you're not using these you can remove them and just leave them as a normal div